src/Contract/Operation/Unwrapable.php

```diff
@@ -18,7 +18,7 @@ interface Unwrapable
      *
      * @see https://loophp-collection.readthedocs.io/en/stable/pages/api.html#unwrap
      *
-     * @return Collection<mixed, mixed>
+     * @return Collection<TKey, T>
```

Comment:
not too sure about this change -> it's not a given that the elements in the unwrapped collection will be all of the same type 🤔 

```PHP
Collection::fromIterable([['a'], [1], [true]])
    ->unwrap(); // ['a', 1, true]
```

unless static analysis is able to interpret the template `T` as the union `string|int|bool` in this case. I'm not sure but you could test this with a static analysis check